File: capstone2/src/main/java/kr/mjc/yoonsungbae/capstone2/SixthScreen.java
package kr.mjc.yoonsungbae.capstone2;
import javax.swing.*;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

public class SixthScreen extends JFrame {
    private String[] exercises = {"푸쉬업", "턱걸이", "파이크 푸쉬업", "레그 레이즈"};
    private int currentExerciseIndex = 0;
    private JLabel exerciseLabel;
    private JButton nextButton;

    public SixthScreen() {
        setTitle("운동 진행 중");
        setSize(800, 1000);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);

        exerciseLabel = new JLabel(exercises[currentExerciseIndex]);
        nextButton = new JButton("다음 운동");

        nextButton.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                nextExercise();
            }
        });

        JPanel panel = new JPanel();
        panel.add(exerciseLabel);
        panel.add(nextButton);
        add(panel);

        setVisible(true);
    }

    private void nextExercise() {
        if (currentExerciseIndex < exercises.length - 1) {
            currentExerciseIndex++;
            exerciseLabel.setText(exercises[currentExerciseIndex]);
        } else {
            new SeventhScreen();
            dispose();
        }
    }
}

File: capstone/src/main/kotlin/SixthScreen.kt
package yoonsungbae

import javax.swing.*

fun createSixthScreen(): JPanel {
    val panel = JPanel()
    panel.layout = BoxLayout(panel, BoxLayout.Y_AXIS)
    panel.border = BorderFactory.createEmptyBorder(20, 20, 20, 20)

    val label = JLabel("진행 상황 및 통계 화면")
    label.font = label.font.deriveFont(18.0f)
    panel.add(label)

    val exerciseLabel = JLabel("푸시업")
    val startButton = JButton("시작")
    val restLabel = JLabel("1분간 휴식하세요")
    val timerLabel = JLabel("01:00")
    val nextButton = JButton("다음 운동")

    startButton.addActionListener {
        JOptionPane.showMessageDialog(panel, "운동 시작")
    }
    nextButton.addActionListener {
        JOptionPane.showMessageDialog(panel, "다음 운동으로 이동")
    }

    panel.add(exerciseLabel)
    panel.add(startButton)
    panel.add(restLabel)
    panel.add(timerLabel)
    panel.add(nextButton)

    return panel
}
